src/bootstrap/mk/Makefile.in

```diff
@@ -45,8 +45,6 @@ check-aux:
 		src/tools/cargo \
 		src/tools/cargotest \
 		$(BOOTSTRAP_ARGS)
-check-bootstrap:
-	$(Q)$(CFG_PYTHON) $(CFG_SRC_DIR)src/bootstrap/bootstrap_test.py
```

Comment:
I think this should be kept except now calling the appropriate subcommand of `$(BOOTSTRAP)`.

Reply:
Never mind, it already runs as part of `./x.py test`.